src/main/java/com/github/pereiratostain/generator/MermaidSchemaGenerator.java

```diff
@@ -33,10 +42,90 @@ private void generateHeader(Writer writer) throws IOException {
   }
 
   private void generateEntity(Writer writer, Entity entity) throws IOException {
+    var associations = new ArrayList<String>();
+    var stereotype = "";
+    String fields;
+
+    if(entity.stereotype() == Entity.Stereotype.ENUM) {
+      fields = computeFieldsEnum(entity);
+      stereotype = "\t<<enumeration>>\n";
+    } else {
+      fields = computeFieldsClass(entity, associations);
+    }
+
     writer.append("    class ")
```

Comment:
given that Writer.append() can be a system call, it's better to gather the value as a String and use few calls to append().

Reply:
Done